README.md

```diff
@@ -2,7 +2,9 @@
 
 The Mbed cryptography library is a reference implementation of the cryptography interface of the Arm Platform Security Architecture (PSA). This is a preview release of Mbed Crypto, provided for evaluation purposes only.
 
-Mbed Crypto is distributed under the Apache License, version 2.0. See the [LICENSE](LICENSE) file for the full text of the license.
+## License
+
+Unless specifically indicated otherwise in a file, Mbed TLS files are provided under the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) license. See the [LICENSE](LICENSE) file for the full text of this license. Contributors must accept that their contributions are made under both the Apache-2.0 AND [GPL-2.0-or-later](https://spdx.org/licenses/GPL-2.0-or-later.html) licenses. This enables LTS (Long Term Support) branches of the software to be provided under either the Apache-2.0 OR GPL-2.0-or-later licenses.
```

Comment:
As discussed earlier, we could try clarifying the inbound licenses a little bit. Maybe writing something like this in addition: "The inbound licences are both Apache-2.0 AND GPL-2.0-or-later simultaneously, that is, contributions are made under both."

Reply:
I think I already clarified that point. Your suggestion just says the same thing twice IMO.